templates/privacy-preferences-modal.php

```diff
@@ -70,7 +70,15 @@
 								<?php foreach ( $args['consent_types'] as $consent_key => $consent_type ) : ?>
 									<div class="gdpr-cookies-used">
 										<div class="gdpr-cookie-title">
-											<p><?php echo esc_html( $consent_type['name'] ); ?></p>
+											<p>
+												<?php if ( $consent_type['policy-page'] > 0 ) : ?>
+													<a href="<?php echo esc_url( get_permalink( $consent_type['policy-page'] ) ); ?>" target="_blank">
```

Comment:
May be worth adding a check that `get_permalink( $consent_type['policy-page'] ) )` doesn't return false

Reply:
Good spot. After doing some testing, I've added more defensive checks.